src/workloads/query/TelemetryWithRedaction.yml

```diff
@@ -0,0 +1,193 @@
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  MaxPhases: &MaxPhases 7
+  nop: &Nop {Nop: true}
+  dbname: &db test
+  binDataGen: &binDataGen {^BinData: {numBytes: 32}}
+
+ActorTemplates:
+- TemplateName: TelemetryAggregation
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "TelemetryAggregation"}}
+    Type: AdminCommand
+    Threads: 32
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1024}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Duration: 1 minute
+          SleepBefore: 5 seconds
+          Database: admin
+          Operations:
+          - OperationMetricsName: TelemetryWithRedactionMetrics
+            OperationName: AdminCommand
+            OperationCommand:
+              aggregate: 1
+              pipeline: [
+                      {
+                        $telemetry: {
+                          redactIdentifiers: {^Parameter: {Name: "redactIdentifiers", Default: false}},
+                          redactionKey: *binDataGen
+                        }
+                      }
+                    ]
+              cursor: {}
+
+Actors:
+- Name: ClearCollection
+  Type: CrudActor
+  Database: *Database
+  Threads: 1
+  Phases:
+    OnlyActiveInPhases:
+      Active: [0]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Threads: 1
+        Collection: *Collection
+        Operations:
+        - OperationName: drop
+
+- Name: InsertData
+  Type: Loader
+  Threads: 4
+  Phases:
+    OnlyActiveInPhases:
+      Active: [1]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+        Database: *Database
+        MultipleThreadsPerCollection: true
+        CollectionCount: 1
+        DocumentCount: *DocumentCount
+        BatchSize: 1000
+        Document:
+          a: &integer {^RandomInt: {min: -100, max: 100}}
+          b: *integer
+          string: &string {^RandomString: {length: 5}}
+        Indexes:
+          - keys: {a: 1, b: 1}
+          - keys: {string: 1}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- Name: FindCommand
+  Type: CrudActor
+  Threads: 32
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [3]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Duration: 1 minute
+        Collection: *Collection
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {string: {$in: ["hello", "hi", "bye"]}}
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *Database
+  Phases:
+    OnlyActiveInPhases:
+      Active: [4]
+      NopInPhasesUpTo: *MaxPhases
+      PhaseConfig:
+        Repeat: 1
+
+- Name: FindCommandWithRedaction
```

Comment:
Can we clarify what "with redaction" means in the name? I think you maybe just mean to say that it is expected to run in the same phase as the telemetry w/ redaction actor?